src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -31,6 +31,12 @@
      */
     META("Meta"),
 
+    /**
+     * KDTree written by {@code BKDWriter} indexes mappings of term to one or more segment row IDs
```

Comment:
```suggestion
     * KDTree written by {@link BKDWriter} indexes mappings of term to one or more segment row IDs
```
Although I think there's going to be some discussion around naming, so this is more a reminder to use `@link`